src/currencyservice/src/server.cpp

```diff
@@ -1,5 +1,16 @@
-// Copyright The OpenTelemetry Authors
-// SPDX-License-Identifier: Apache-2.0
+ // Copyright The OpenTelemetry Authors
+ //
```

Comment:
Fix the indentation here:

```suggestion
//
```